scripts/create-change-file.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+ 
```

Comment:
@dzearing we should make these changes directly in NPMX.. I'm going to create a tracking branch in sp-client.